File: Feitos em C Sharp/2598.cs
using System;
namespace ConsoleApp26 {
    class Program {
        static void Main(string[] args) {
			int a = int.Parse(Console.ReadLine());
			for (int i = 0; i < a; i++) {
				int quant = 0;
				string[] vet = Console.ReadLine().Split();
				int n = Convert.ToInt32(vet[0]);
				int m = Convert.ToInt32(vet[1]);
				while (n > 0) {
					n -= m;
					quant++;
				}
				Console.WriteLine(quant);
			}
		}
    }
}

File: Feitos em Python 3/2598.py
a=int(input())
for i in range(0,a):
    vet=input().split()
    n=int(vet[0])
    m=int(vet[1])
    quant=0
    while n>0:
        n-=m
        quant+=1
    print(quant)